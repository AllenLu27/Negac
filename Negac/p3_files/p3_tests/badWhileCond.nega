void fn(){
	while (1){ }
	while (a){ }
}
