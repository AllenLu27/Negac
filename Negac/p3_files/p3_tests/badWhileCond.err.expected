FATAL [3,9]: Undeclared identifier
FATAL [2,9]: Non-bool expression used as a while condition
Semantic analysis failed
